discovery_engine/pubspec.yaml

```diff
@@ -8,27 +8,27 @@ environment:
   sdk: '>=2.14.2 <3.0.0'
 
 dependencies:
-  crypto: 3.0.1
-  equatable: 2.0.3
-  freezed_annotation: 0.15.0
-  hive: 2.0.4
-  http: 0.13.4
-  http_retry: 0.2.0
-  json_annotation: 4.3.0
-  logger: 1.1.0
-  meta: 1.7.0
-  universal_platform: 1.0.0+1
-  uuid: 3.0.5
+  crypto: ^3.0.1
+  equatable: ^2.0.3
+  freezed_annotation: ^1.1.0
+  hive: ^2.0.5
+  http: ^0.13.4
+  http_retry: ^0.2.0
+  json_annotation: ^4.4.0
+  logger: ^1.1.0
+  meta: ^1.7.0
+  universal_platform: ^1.0.0+1
+  uuid: ^3.0.5
```

Comment:
We wanted to use exact version, as we don't track lock file in git. What is the reason for starting with using caret?

Reply:
ah, ok.. I've just read the PR description. Maybe we should then start tracking the lockfile? Not sure what is the best solution here..

Reply:
To give more freedom to blue to use different versions. We can start using the lock and not exporting it on the release repo if we want.